sodetlib/operations/bias_dets.py

```diff
@@ -78,21 +79,35 @@ def bias_to_rfrac_range(
     for bg in bias_groups:
         S.log(f"BG {bg}: {biases[bg]:.2f}")
 
-    S.log("Overbiasing detectors")
-    sdl.set_current_mode(S, bias_groups, 1)
-    for bg in bias_groups:
-        S.set_tes_bias_bipolar(bg, overbias_voltage)
-    time.sleep(overbias_wait)
+    if overbias:
+        sdl.overbias_dets(S, cfg, bias_groups=bias_groups)
     S.set_tes_bias_bipolar_array(biases)
-    sdl.set_current_mode(S, bias_groups, 0, const_current=False)
 
     return biases
 
+def bias_to_volt_arr(S, cfg, biases, bias_groups=None, overbias=True):
+    if bias_groups is None:
+        bias_groups = cfg.dev.exp['active_bgs']
+    bias_groups = np.atleast_1d(bias_groups)
+
+    _biases = S.get_tes_bias_bipolar_array()
+    for bg in bias_groups:
+        _biases[bg] = biases[bg]
+
+    S.log(f"Target biases: ")
+    for bg in bias_groups:
+        S.log(f"BG {bg}: {biases[bg]:.2f}")
+
+    if overbias:
+        sdl.overbias_dets(S, cfg, bias_groups=bias_groups)
+
+    S.set_tes_bias_bipolar_array(biases)
+    return
+
```

Comment:
Ya this was not clear without a docstring, but this function expects biases to be an array of length 12, with `biases[bg]` to be the bias of bias line `bg`. I made it like this since I found it easier to work with / I was starting with the array returned by `S.get_tes_bias_bipolar_array` and modifying it. New docstrings should clear this up